components/cloak-visual.coffee

```diff
@@ -104,7 +104,8 @@ export default
 		# Disable lazy loading automatically if found in 2 blocks. Written
 		# kinda weird so it defaults to true when blockIndex is undefined
 		isCriticalImage = injections.blockIndex < 2
-		lazyload = props.lazyload ? not isCriticalImage
+		preload = props.preload ? isCriticalImage
```

Comment:
You'll need to add a `preload` prop [like this](https://github.com/BKWLD/cloak-visual/blob/1e9ccf0ee0fa9db669b24d0ad07eba640a8d9e61/components/visual.coffee#L43-L45) too, otherwise `props.preload === false`.  We want it to `=== undefined` for use in this ` ? ` test.

Reply:
`preload` is part of `vue-visual` [props](https://github.com/BKWLD/vue-visual/blob/master/concerns/supports-images.coffee#L10) since merging the support [preload PR](https://github.com/BKWLD/vue-visual/pull/76), and `cloak-visual` reduces  `vue-visual` mixin's props and [inherits those props](https://github.com/BKWLD/cloak/pull/63/files/cde632ab778c505bcac31a202a380b8b9800b656#diff-d8e7810cdaeec713ccc3cc13143336abb263431e9ca81325b93a4e33e6872fa1R9-R23), so `props.preload` should be `undefined` if no `preload` value is passed as prop.

Reply:
Well, the prob is that in `vue-visual`, it's a straight up Boolean prop without an explicit default, so it defaults to `false` when undefined.  So without overriding the prop to have a an _explicit_ default of `undefined`, `props.preload` will  `=== false` whenever it's not specified.

Reply:
Yeah you are right, it does default to `false`. I believed that every defined prop without a set default would take `undefined` if no value was passed, but that's not the case with Boolean props. Is it the only type that behaves like this? Just pushed a change adding a `preload` that defaults to undefined.

Reply:
Yeah, I think it's just `Boolean` that has this kinda behavior.  Maybe `String` defaults to `""`?  Not 💯 about that.